bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/io/ReconnectingChannel.java

```diff
@@ -120,29 +153,54 @@ public void sendMessage(RequestT message) {
       Preconditions.checkState(callDelegate != null, "Not started");
       callDelegate.sendMessage(message);
     }
-
   }
 
-  // We can't do a newCall on a closed delegate.  This will ensure that refreshes don't 
-  // allow a closed delegate to perform a newCall.  Once closed is called, all existing
-  // calls will complete before the delegate shuts down.
-  private final AtomicInteger closingAsynchronously = new AtomicInteger(0);
-
   private final long maxRefreshTime;
   private final Factory factory;
+  private final String authority;
+
+  private ChannelWrapper delegate;
+
+  public ReconnectingChannel(long maxRefreshTime, Factory connectionFactory) throws IOException {
```

Comment:
Seems like this could use some javadoc. In particular, what's the time unit for `maxRefreshTime`?